src/Forms/RemoveTextFromHearImpaired.cs

```diff
@@ -282,7 +282,7 @@ public RemoveTextForHISettings GetSettings(Subtitle subtitle)
                 RemoveIfTextContains = new List<string>(),
                 RemoveTextBetweenCustomTags = checkBoxRemoveTextBetweenCustomTags.Checked,
                 RemoveInterjections = checkBoxRemoveInterjections.Checked,
-                RemoveInterjectionsOnlySeparateLine = checkBoxInterjectionOnlySeparateLine.Checked,
+                RemoveInterjectionsOnlySeparateLine = checkBoxRemoveInterjections.Checked && checkBoxInterjectionOnlySeparateLine.Checked,

```

Comment:
only remove interjection in separated lines if "remove interjection" is activated